stdlib/LinearAlgebra/test/symmetriceigen.jl

```diff
@@ -8,7 +8,7 @@ using Test, LinearAlgebra
     ## Cholesky decomposition based
 
     # eigenvalue sorting
-    sf = x->(real(x),imag(x))
+    sf = x->(imag(x),real(x))
```

Comment:
Why changing the sorting function in unrelated tests?

Reply:
Thank you for this comment. Indeed, this change looks weird, but is unfortunately necessary. I discuss it in the post below.